ricochet.pro

```diff
@@ -106,21 +106,26 @@ win32-msvc2008|win32-msvc2010 {
 
 INCLUDEPATH += src
 
+win32|mac {
+    # For mac, this is necessary because homebrew does not link openssl .pc to
+    # /usr/local/lib/pkgconfig (presumably because it used to be a system
+    # package).
+    #
+    # Unfortunately, it is no longer really a system package, and we really
+    # need to know where it is.
+    isEmpty(OPENSSLDIR): error(You must pass OPENSSLDIR=path/to/openssl to qmake on this platform)
```

Comment:
This does unfortunately mean that macs with a valid `pkg-config libcrypto` won't use it. But testing `packagesExists(libcrypto)` would be insufficient because macOS' old 0.9.8 seems to leave behind some broken pkgconfig files (and because of the no-pkg-config problem).

So, the workaround is annoying and I'm ok with this for the current state of things.